docs/2.10.0/docs/canton/usermanual/packagemanagement.rst

```diff
@@ -61,235 +61,183 @@ Understanding Package Vetting
+must upload and enable vetting for the same DAR and its contained packages before the transaction is submitted.
 
-If one of the involved participants doesn't know about a certain DAR, then the transaction will bounce with an error
+If one of the involved participants doesn't know about the packages of a certain DAR, then the transaction will bounce with an error
 ``NO_DOMAIN_FOR_SUBMISSION`` (with additional metadata listing which package has not been vetted on which participant).
-
-This error goes back to the fact that both participants not only upload the DAR, but also publicly declare towards
-their peers that they are ready to receive transactions referring to certain packages. This declaration happens
-automatically when you upload a DAR. The package vettings can be inspected using (preview):
-
-.. snippet:: package_dar_removal
-    .. success(output=8):: participant2.topology.vetted_packages.list()
-
-Vetting is necessary, as otherwise, a malicious participant might send a transaction referring to package a receiver
-does not have, which would make it impossible for the receiver to process the transaction, leading to a ledger fork.
-As transactions are valid only if all involved participants have vetted the used packages, this attack cannot happen.
-
-Removing Packages and DARs
---------------------------
+This can happen if the DAR has not been uploaded to the participant node or it has not been vetted.
 
 .. note::
+    The unit of Daml code for which a Canton participant can issue topology transactions is the Daml package.
+    However, in practice, for ensuring that transactions can be successfully submitted using any package of a specific DAR,
+    the high-level Canton vetting APIs act on vetting at the DAR level (i.e. for all of a DAR's packages, transactionally).
+    These high-level APIs are the ``participant.dars.vetting.enable`` and ``participant.dars.vetting.disable`` console commands as well
+    as the ``PackageService.vetDar`` and ``PackageService.unvetDar`` Admin API calls.
 
-    Note that package and DAR removal is under active development. The behaviour described in this documentation may
-    change in the future. Package and DAR removal is a preview feature and should not be used in production.
-
-Canton supports removal of both packages and DARs that are no longer in use. Removing unused packages and DARs has the following advantages:
-
-- Freeing up storage
-
-- Preventing accidental use of the old package / DAR
-
-- Reducing the number of packages / DARs that are trusted and may potentially have to be audited
+Vetting happens automatically when you upload a DAR, although it can be disabled by a request flag.
 
+.. snippet:: package_dar_vetting
+    .. success:: participant2.dars.upload("dars/CantonExamples.dar", vetAllPackages = false)
 
+Vetting can also be enabled explicitly for a DAR's packages (e.g. if the DAR upload operation was triggerred without vetting)
```

Comment:
Same here: should we say that we vet packages rather than enabling vetting for packages ?